adoc/deployment-loadbalancer.adoc

```diff
@@ -70,17 +70,31 @@ stream {
         proxy_timeout 3s;
         proxy_pass k8s-masters;
     }
+
+    server {
+      listen 32000; // <3>
+      proxy_pass master00:32000; // <2>
```

Comment:
Gangway and Dex are exported by NodePort, they are able to access from all nodes. I would suggest to use the same settings as API server as example.

cc @jenting

Reply:
I changed settings for Dex and Gangway. 
@innobead Thanks for the code review.

Reply:
This is not I suggested :) 

Like we discussed, you just need to create similar settings of k8s-masters for dex-backends and gangway-backends, then use the correct port (32000, 32001).

Reply:
If `master00` down, is dex + gangway still accessible?

Reply:
Somehow, accidentally I rolled back to old version. 
I reverted back to new version.